packages/cli/src/rpc/explain/location.ts

```diff
@@ -34,19 +51,21 @@ export default class Location {
   }
 
   static parse(location: string): Location {
-    const tokens = location.split(':');
```

Comment:
I forgot to mention this in the PR description, but splitting here on `:` is also problematic for absolute paths in Windows. I've rewritten this method to perform the path and line range parsing via regex.